pkg/cvo/metrics.go

```diff
@@ -83,6 +103,10 @@ version for 'cluster', or empty for 'initial'.
 			Name: "cluster_installer",
 			Help: "Reports info about the installation process and, if applicable, the install tool.",
 		}, []string{"type", "version", "invoker"}),
+		clusterVersionOperatorSuccessfulUpdateRetrieval: prometheus.NewGaugeVec(prometheus.GaugeOpts{
+			Name: "cluster_version_operator_successful_update_retrieval",
```

Comment:
better to use `cluster_version_operator_update_retrieval_timestamp_seconds` to follow metric naming [conventions](https://prometheus.io/docs/practices/naming/#metric-names).

Reply:
Done as of 06c550ff9fed.